ios/Handlers/ChangePinHandler.swift

```diff
@@ -0,0 +1,45 @@
+protocol PinConnectorToPinHandler: AnyObject {
+    func onChangePinCalled(completion: @escaping (Bool, NSError?) -> Void)
+}
+
+enum PINEntryMode {
+    case login
+    case registration
+    
+}
+
+class ChangePinHandler: NSObject {
+    var flow: PinFlow?
+    var changePinCompletion: ((Bool, NSError?) -> Void)?
```

Comment:
can we use Error here, so you won't need to cast error as NSError in other places?